test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -79,13 +78,8 @@ static void test(Path top) throws IOException {
         Instant now = Instant.now();
         if (Math.abs(creationTime.toMillis()-now.toEpochMilli()) > 10000L) {
             System.out.println("creationTime.toMillis() == " + creationTime.toMillis());
```

Comment:
Sorry but would you please change line 80 to use `System.err` also? Thanks.

Reply:
Sorry for miss the replaced. The line 80 has been changed.